BitwardenShared/Core/Platform/Models/Domain/EnvironmentUrlData.swift

```diff
@@ -118,8 +133,24 @@ extension EnvironmentUrlData {
 
 extension EnvironmentUrlData {
     /// The default URLs for the US region.
-    static let defaultUS = EnvironmentUrlData(base: URL(string: "https://vault.bitwarden.com")!)
+    static let defaultUS = EnvironmentUrlData(
+        api: URL(string: "https://api.bitwarden.com")!,
+        base: URL(string: "https://vault.bitwarden.com")!,
+        events: URL(string: "https://events.bitwarden.com")!,
+        icons: URL(string: "https://icons.bitwarden.net")!,
+        identity: URL(string: "https://identity.bitwarden.com")!,
+        notifications: URL(string: "https://notifications.bitwarden.com")!,
+        webVault: URL(string: "https://vault.bitwarden.com")!
+    )
 
     /// The default URLs for the EU region.
-    static let defaultEU = EnvironmentUrlData(base: URL(string: "https://vault.bitwarden.eu")!)
+    static let defaultEU = EnvironmentUrlData(
+        api: URL(string: "https://api.bitwarden.eu")!,
+        base: URL(string: "https://vault.bitwarden.eu")!,
+        events: URL(string: "https://events.bitwarden.eu")!,
+        icons: URL(string: "https://icons.bitwarden.net")!,
```

Comment:
❓ Based on the MAUI code you linked, should this be `https://icons.bitwarden.eu`?

https://github.com/bitwarden/mobile/blob/main/src/Core/Models/Data/EnvironmentUrlData.cs#L26C22-L26C48

Reply:
Nice catch, I'll change that one!